### PR TITLE
Feature/sonarlint/include 4

### PR DESCRIPTION
**Pull request by PedroErnst**
Indentation and Reformat
Fix spacing around control structures
Break over sized lines
Missing braces
Fix PHPDocs
Remove unused variables and dead code
Explicitly state method and variable visibility

## Motivation and Context

Make the code easier to read and to maintain.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: include/UploadFile.php [hunk 1]
```diff
@@ -51,32 +51,63 @@
...
-
     /**
      * Upload errors
      * @var array
      */
     protected static $filesError = array(
         UPLOAD_ERR_OK => 'UPLOAD_ERR_OK - There is no error, the file uploaded with success.',
-        UPLOAD_ERR_INI_SIZE => 'UPLOAD_ERR_INI_SIZE - The uploaded file exceeds the upload_max_filesize directive in php.ini.',
-        UPLOAD_ERR_FORM_SIZE => 'UPLOAD_ERR_FORM_SIZE - The uploaded file exceeds the MAX_FILE_SIZE directive that was specified in the HTML form.',
+        UPLOAD_ERR_INI_SIZE => 'UPLOAD_ERR_INI_SIZE - The uploaded file exceeds the upload_max_filesize
```
Comment (horus68) [inline include/UploadFile.php]: Why breaking language lines?

Comment (PedroErnst) [reply include/UploadFile.php]: Lines should be no longer than 120 characters.

On the other hand, this doesn't affect the result, the line break will not be included in the resulting string.

And yes those seem to be hard coded. But I don't know if they are actually being used anywhere or if that is just dead code...

Comment (horus68) [reply include/UploadFile.php]: Searching for it here: https://github.com/salesagility/SuiteCRM/search?utf8=%E2%9C%93&q=UPLOAD_ERR_INI_SIZE&type=
Note: line breaks "WHEN" in language files do matter as the translation platform makes that a real line break and makes the task harder for translators. Not the case here as this is not a language file.

Comment (PedroErnst) [reply include/UploadFile.php]: Thanks for the heads up regarding line breaks in language files.

**Pull request closed without merging**